src/Sylius/Bundle/AttributeBundle/Controller/AttributeController.php

```diff
@@ -48,6 +48,8 @@ public function getAttributeTypesAction(Request $request, $template)
      */
     public function renderAttributesAction(Request $request)
     {
+        $template = ($request->attributes->has('template')) ? $request->attributes->get('template') : 'SyliusAttributeBundle::attributeChoice.html.twig';
```

Comment:
third argument in ``->get()`` method is the default value so you can use it instead of this ``if`` statement.